src/utils/components/withErrorList.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import withErrorList, { getErrorId } from './withErrorList';
+import ErrorList from '../../forms/ErrorList';
+import { mount } from 'enzyme';
+
+const MOCK_FIELD_ID = 'name';
+const MOCK_ERROR_MSG = 'You are not Keith Hernandez';
+
+const TestComponent = (props) => (<h1 {...props}>Hello world</h1>);
+const TestComponentWithErrorList = withErrorList(TestComponent);
+
+describe('error id', () => {
+	it('correctly generates id for `aria-describedby` from field `id` prop', () => {
+		const actual = getErrorId(MOCK_FIELD_ID);
+		const expected = 'name-error';
+		expect(actual).toBe(expected);
+	});
+});
+
+describe('withErrorList wrapped component', () => {
+
+	it('passes `error` to the `ErrorList`', () => {
+		const component = mount(
```

Comment:
do you need `mount` here? `shallow` should cover anything that the HOC is responsible for - e.g. the HOC doesn't specify that the `<ErrorList>` needs to use an `li` element to display the error, so this test doesn't need to look for `li`. Instead, just validate that the props for `ErrorList` are the expected values (or take a snapshot)